devices/tuya/_TZE200_TS0601_water_valve.json

```diff
@@ -0,0 +1,195 @@
+{
+	"schema": "devcap1.schema.json",
+	"manufacturername": ["_TZE200_a7sghmms", "_TZE200_sh1btabb"],
+	"modelid": ["TS0601", "TS0601"],
+	"vendor": "GIEXperience",
+	"product": "Smart water valve",
+	"sleeper": false,
+	"status": "Gold",
+	"subdevices": [{
+			"type": "$TYPE_ON_OFF_OUTPUT",
+			"restapi": "/lights",
+			"uuid": [
+				"$address.ext",
+				"0x01"
+			],
+			"items": [{
+					"name": "attr/id"
+				},
+				{
+					"name": "attr/lastannounced"
+				},
+				{
+					"name": "attr/lastseen"
+				},
+				{
+					"name": "attr/manufacturername"
+				},
+				{
+					"name": "attr/modelid"
+				},
+				{
+					"name": "attr/name"
+				},
+				{
+					"name": "attr/swversion",
+					"parse": {"fn": "zcl:attr", "ep": 1, "cl": "0x0000", "at": "0x0001", "script": "tuya_swversion.js"},
+				        "read": {"fn": "zcl:attr", "ep": 1, "cl": "0x0000", "at": "0x0001"}
+				},
+				{
+					"name": "attr/type"
+				},
+				{
+					"name": "attr/uniqueid"
+				},
+				{
+					"name": "state/on",
+					"refresh.interval": 300,
+					"read": {
+						"fn": "tuya"
+					},
+					"write": {
+						"dpid": 2,
+						"dt": "0x10",
+						"eval": "Item.val == 1 ? 1 : 0;",
+						"fn": "tuya"
+					},
+					"parse": {
+						"dpid": 2,
+						"eval": "Item.val = Attr.val;",
+						"fn": "tuya"
+					},
+					"default": 0
+				},
+				{
+					"name": "config/tuya_unlock"
+				},
+				{
+					"name": "state/reachable"
+				}
+			]
+		},
+		{
+			"type": "$TYPE_CONSUMPTION_SENSOR",
+			"restapi": "/sensors",
+			"uuid": [
+				"$address.ext",
+				"0x01",
+				"0x0702"
+			],
+			"items": [{
+					"name": "attr/id"
+				},
+				{
+					"name": "attr/lastannounced"
+				},
+				{
+					"name": "attr/lastseen"
+				},
+				{
+					"name": "attr/manufacturername"
+				},
+				{
+					"name": "attr/modelid"
+				},
+				{
+					"name": "attr/name"
+				},
+				{
+					"name": "attr/swversion",
+					"parse": {"fn": "zcl:attr", "ep": 1, "cl": "0x0000", "at": "0x0001", "script": "tuya_swversion.js"},
+				        "read": {"fn": "zcl:attr", "ep": 1, "cl": "0x0000", "at": "0x0001"}
+				},
+				{
+					"name": "attr/type"
+				},
+				{
+					"name": "attr/uniqueid"
+				},
+				{
+					"name": "config/battery",
+					"parse": {
+						"dpid": 108,
+						"eval": "Item.val = Attr.val;",
+						"fn": "tuya"
+					},
+					"default": 0
+				},
+				{
+					"name": "config/duration",
+					"description": "Last irrigation duration",
+					"parse": {
+						"dpid": 114,
+						"eval": "Item.val = Attr.val;",
+						"fn": "tuya"
+					}
+				},
+				{
+					"name": "state/consumption",

```

Comment:
Using ZHAConsumption's for water consumption (rather than electricity) is new for me, but seems logical.  However, I don't think it's a good idea to abuse `state/consumption`.  Instead, we should be using `state/measured_value` and the corresponding `cap` items to indicate the substance (water), quantity (volume), and unit (litres or m3?), as well as min/max values.  
Typically `state/consumption` reports the life time consumption (since last factory reset), not the consumption during last run.